index.html

```diff
@@ -12,7 +12,7 @@
     <link rel="icon" type="image/png" href="images/YOSHITHA RATHNAYAKE PRO.png" />
 </head>
 <body>
-    <h1>YOSHITHA RATHNAYAKE PRO</h1>
+    <h1 style="text-align:center">YOSHITHA RATHNAYAKE PRO</h1>
```

Comment:
Great! but some changes requested. Can you remove this inline css and wrap the h1 tag with a `<center>` tag @atharvabhoite7

Reply:
@YoshithaRathnayake  I have done the required changes.  Please review them.